### DISCUSSION TITLE
About FIle Upload component, how to send file ?

### DISCUSSION THREAD

**Discussion by Asteb612** [chakra-ui/ark#3071] [Q&A] [answered]
On the documentation i haven't found anything about the file upload. Do you have some example of file upload with form and without form ? Does this is related to the `useFileUpload` ? I ask because i haven't found anything that help me to understand how it works.

**Comment by cschroeter** [comment, marked as answer]
@Asteb612 

Hey mate thanks for reaching out. The `FileUpload` component in Ark is primarily there to help your validate user input such as file size, type etc. Uploading a file to a server is something that happens on user land. for example here is how you could start:

```tsx
import { FileUpload } from '@ark-ui/react/file-upload'

const Demo = () => {
  const handleFileAccept = (files: FileUpload.FileAcceptDetails) => {
    // send files to server
  }
  return (
    <FileUpload.Root onFileAccept={handleFileAccept} maxFiles={5}>
    // ...
    </FileUpload.Root>
    )
  }

**Reply by Asteb612** [reply]
Thanks a lot, i think adding this exemple will be really helpful to other.  Because without your answer i was absolutely lost.
